docs/reference/components/sources/postgresql_metrics.cue

```diff
@@ -0,0 +1,391 @@
+package metadata
+
+components: sources: postgresql_metrics: {
+	title:       "PostgreSQL Metrics"
+	description: "[PostgreSQL][urls.postgresql] PostgreSQL is a powerful, open source object-relational database system with over 30 years of active development that has earned it a strong reputation for reliability, feature robustness, and performance."
+
+	classes: {
+		commonly_used: false
+		delivery:      "at_least_once"
+		deployment_roles: ["daemon", "sidecar"]
+		development:   "beta"
+		egress_method: "batch"
+	}
+
+	features: {
+		collect: {
+			checkpoint: enabled: false
+			from: {
+				service: {
+					name:     "PostgreSQL Server"
+					thing:    "an \(name)"
+					url:      urls.postgresql
+					versions: null
```

Comment:
Do you know if there are any restrictions on Postgres versions with the queries we are making? It might be prudent to just call out all currently supported PostgreSQL versions here (9-13) assuming our queries work for all of them.

https://www.postgresql.org/support/versioning/

Reply:
Good question! Checked and adjusted code for `9.6-13`.